great_expectations/core/batch.py

```diff
@@ -13,6 +15,14 @@
 
 logger = logging.getLogger(__name__)
 
+try:
+    import pyspark
+except ImportError:
+    pyspark = None
+    logger.debug(
```

Comment:
I'd have suspected this would be a 'warn' level, but I trust you call on this. We don't want unnecessary noisy stdouts

Reply:
I was just following pattern in rest of GE.